installation/cmd/run.sh

```diff
@@ -275,6 +264,12 @@ function patchJWKS() {
 }
 patchJWKS&
 
+echo 'Installing DB'
```

Comment:
The doc for OSS installation should also reflect that change: 

There install-compass.sh is executed directly, we need to document that install-db should be executed as a prerequisite.